### PR TITLE
New Blog for Release KubeEdge v1.15

### LINKED ISSUE #576: Sync v1.15  changelog(what's new part) to the blog
Now in blog, https://kubeedge.io/blog, we have no blog for v1.15, so plz sync them from the changelog(what's new part) :)

changelog: https://github.com/kubeedge/kubeedge/tree/master/CHANGELOG
example: https://github.com/kubeedge/website/pull/517

### PR DESCRIPTION
**Pull request by 1Shubham7**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
docs update

* **What is the new behavior (if this is a feature change)?**
Added a new Blog for Release 1.15, closes #576

### CODE REVIEW

File: blog/release-v1.15/index.mdx [hunk 1]
```diff
@@ -0,0 +1,99 @@
+---
+authors:
+- KubeEdge SIG Release
+categories:
+- General
+- Announcements  
+date: 2023-03-31
```
Comment (Shelley-BaoYue) [inline blog/release-v1.15/index.mdx:7]:
```suggestion
date: 2023-10-13
```

File: blog/release-v1.15/index.mdx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+draft: false
+lastmod: 2023-03-31
```
Comment (Shelley-BaoYue) [inline blog/release-v1.15/index.mdx:9]:
```suggestion
lastmod: 2023-10-13
```

**Comment by 1Shubham7** [comment]
sorry for messing up the dates :(

**Review by fisherxu — COMMENTED** [review]
/approve

**Review by Shelley-BaoYue — COMMENTED** [review]
/lgtm

**Pull request merged**